src/Subsystems/Turret.cpp

```diff
@@ -51,15 +54,25 @@ double Turret::GetSetPoint() const {
 }
 
 double Turret::GetTurretPosition() const {
-	return turretSpinner.GetEncPosition();
+  return turretSpinner.GetPosition();
 }
 
-bool Turret::IsLeftSwitchPressed() const{
+bool Turret::IsLeftStopperHit() const{
   return turretSpinner.IsFwdLimitSwitchClosed();
 }
 
-bool Turret::IsRightSwitchPressed() const{
+bool Turret::IsRightStopperHit() const{
   return turretSpinner.IsRevLimitSwitchClosed();
 }
+
+void Turret::Calibrate(){
+  turretSpinner.SetControlMode(CANTalon::kPercentVbus);
+}
+
+void Turret::FinishCalibrate(double newPosition){
+  turretSpinner.SetControlMode(CANTalon::kPosition);
+  defaultCommand->SetPosition(newPosition);
+  SetTurretPosition(newPosition);
```

Comment:
We'll need to check if these are in the right order. We'll see if the turret does anything odd when the calibration is set.